frontend/src/components/Note.jsx

```diff
@@ -23,7 +24,11 @@ export default function Note({ role, id, group, user, refetchGroup }) {
   };
 
   async function loadNote() {
-    const noteData = await usePost("/api/note/retrieve", { noteId: id });
+    const noteData = await usePost(
+      "/api/note/retrieve",
+      { noteId: id },
+      getUserIdToken
```

Comment:
Unsafe usage of `getUserIdToken` - have a read of https://wdcc-vps.atlassian.net/wiki/spaces/W/pages/14417933/Auth+Implementation+Guide

notably the user stateful value and the other error and loading cases.

Reply:
This is still unsafe right? What if `getUserIdToken` returns a null result? What then? What if it's simply authenticating again and it will eventually return a value? What if the authentication fails for whatever reason?